src/blockdata/constants.rs

```diff
@@ -176,6 +176,33 @@ pub fn genesis_block(network: Network) -> Block {
     }
 }
 
+// Mainnet value can be verified at https://github.com/lightning/bolts/blob/master/00-introduction.md
```

Comment:
Is this even part of Bitcoin Core? From my understanding this is a LN and DLC thing.

Reply:
The Genesis block? I think that's a Bitcoin thing :)

Reply:
lol, I meant that chain hash is a LN/DLC thing and the LN spec says that the identifier for mainnet is the hash of the genesis block - hence I linked to the LN spec.

@sanket1729 would you prefer that this comment explained what the value used is but then linked to the bitcoin core source for the actual value?

Reply:
I am with without a comment. But I have a slight preference to link to the bitcoin core source. When reading a constant `GENESIS_BLOCK_HASH_BITCOIN` in file `src/blockdata/constants` in `rust-bitcoin`, I would expect it to be a bitcoin thing. It would feel odd that I need to check a lightning spec to double-check that the bitcoin genesis block is correct. 

ChainHash functions can link to lightning spec if needed.